runtime/thread/CMakeLists.txt

```diff
@@ -44,6 +44,8 @@ endif()
 get_target_property(thread_exports j9thr_obj EXPORTED_SYMBOLS)
 omr_add_exports(j9thr ${thread_exports})
 
+target_enable_ddr(j9thr j9ddr EARLY_SOURCE_EVAL)
```

Comment:
Same comment required here

Reply:
Will add comment, but for the record
https://github.com/eclipse/omr/commit/169003e4378d8799c7ca7d0c41f460fb0049fb2d

Reply:
So this is related to the use of `$<TARGET_OBJECTS:j9thr_obj>`?  It would be really good to have that explained explicitly in this file